src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/cosmic/CosmicFuncotationFactory.java

```diff
@@ -369,18 +380,51 @@ private final SimpleInterval parseProteinString(final String proteinPositionStri
 
         final Matcher matcher = PROTEIN_POSITION_REGEX.matcher(proteinPositionString);
         if ( matcher.matches() ) {
-            // We have a position, so we should parse it.
-            // If we have an end position, we should use it:
-            if ( matcher.group(2) != null ) {
-                return new SimpleInterval(PROTEIN_CONTIG, Integer.valueOf(matcher.group(1)), Integer.valueOf(matcher.group(2)));
+            try {
+
+                // We have a position, so we should parse it.
+                // If we have an end position, we should use it:
+                if ( matcher.group(2) != null ) {
+                    return new SimpleInterval(PROTEIN_CONTIG, Integer.valueOf(matcher.group(1)), Integer.valueOf(matcher.group(2)));
+                }
+                else {
+                    return new SimpleInterval(PROTEIN_CONTIG, Integer.valueOf(matcher.group(1)), Integer.valueOf(matcher.group(1)));
+                }
             }
-            else {
-                return new SimpleInterval(PROTEIN_CONTIG, Integer.valueOf(matcher.group(1)), Integer.valueOf(matcher.group(1)));
+            catch (final IllegalArgumentException ex) {
+                // If we have poorly bounded protein changes, we need to warn the user and move on.
+                // These occur occasionally in the data.
+                logger.warn("Warning - unable to parse protein string due to invalid position information.  Ignoring potential COSMIC match with protein sequence: " + proteinPositionString);
+                return null;
             }
         }
         return null;
     }
 
+    /**
+     * Print the given {@link ResultSet} to stdout.
+     * @param resultSet The {@link ResultSet} to print.
```

Comment:
Why do we need this method?  Typically, we should log it (as a debug if necessary)